src/Components/Cards/Card/BigCreditCard.tsx

```diff
@@ -0,0 +1,14 @@
+import CardLayout from "../CardElements/Layout/CardLayout";
+import BigCardLayout from "../CardElements/Layout/BigCardLayout";
+import CardBody from "../Body/CardBody";
+const BigCreditCard = () => {
```

Comment:
이런 부분은 prettier를 활용해봐도 좋을 것 같네요:)

```suggestion
import CardBody from "../Body/CardBody";

const BigCreditCard = () => {
```